### PR TITLE
Updated State to track when the ad is fully completed

### PR DESCRIPTION
**Pull request by balavor**
## Changes
Added property `isCompleted` to state in `AdFinishTracker` component to know when the ad is completed.

@VerizonAdPlatforms/mobile-sdk-developers: Please review.

### CODE REVIEW

File: PlayerCore/components/AdFinishTracker.swift [hunk 1]
```diff
@@ -5,18 +5,21 @@ import Foundation
 
 public struct AdFinishTracker {
     public let isFinished: Bool
+    public let isCompleted: Bool
```
Comment (VladyslavAnokhin) [inline PlayerCore/components/AdFinishTracker.swift]: Bad naming
`isFinished ` -> `isForceFinished`
`isCompleted` -> `isSuccessfullyCompleted `

Comment (RomanTysiachnik) [reply PlayerCore/components/AdFinishTracker.swift]: Well, seems ok

File: PlayerCore/components/AdFinishTracker.swift [hunk 2]
```diff
@@ -4,19 +4,22 @@
...
+    public let isSuccessfullyCompleted: Bool
 }
 
 func reduce(state: AdFinishTracker, action: Action) -> AdFinishTracker {
     switch action {
     
-    case is ShowAd, is AdMaxShowTimeout:
-        return AdFinishTracker(isFinished: false)
+    case is ShowAd:
+        return AdFinishTracker(isForceFinished: false, isSuccessfullyCompleted: false)
```
Comment (AndriiMoskvin) [inline PlayerCore/components/AdFinishTracker.swift]: What the meaning of `forceFinish`

Comment (RomanTysiachnik) [reply PlayerCore/components/AdFinishTracker.swift]: Means that the ad wasn't completed due to some reason (max show timeout, skip, stop etc.)

**Pull request merged**